fs/copy.ts

```diff
@@ -190,17 +190,21 @@ async function copyDir(
   src = toPathString(src);
   dest = toPathString(dest);
 
+  const promises = [];
+
   for await (const entry of Deno.readDir(src)) {
     const srcPath = join(src, entry.name);
     const destPath = join(dest, basename(srcPath as string));
     if (entry.isSymlink) {
-      await copySymLink(srcPath, destPath, options);
+      promisesCopyAsyncFunctions.push(copySymLink(srcPath, destPath, options));
```

Comment:
```suggestion
      promises.push(copySymLink(srcPath, destPath, options));
```
Sorry, I should've mentioned that these also need to be renamed.